### PR TITLE
feat: add validation for commercetools [MONET-1737]

### PR DESCRIPTION
**Pull request by cemreyuksel**
## Purpose

[Ticket](https://contentful.atlassian.net/browse/MONET-1737)

## Approach

When the API URL contains a trailing / the 3rd-party API request in graph-api fails. This PR removes the '/' if there is one, inside `validateParameters` function. 

## Testing steps

I tested locally by running this app. Only when you refresh the page, the value on the form changes but I think this is not a problem for the bug fix. There were no tests for the application.

### CODE REVIEW

**Review by kdamball — APPROVED** [review]

**Review by kdamball — COMMENTED** [review]
Can we make sure it doesn't break the frontend app

File: apps/commercetools-without-search/src/config.ts [hunk 1]
```diff
@@ -79,5 +79,9 @@ export function validateParameters(parameters: ConfigurationParameters): string
     return 'Provide the commercetools data locale.';
   }
 
+  if (parameters.authApiEndpoint?.endsWith('/')) {
+    parameters.apiEndpoint = parameters.apiEndpoint?.slice(0, -1);
```
Comment (mrlaessig) [inline apps/commercetools-without-search/src/config.ts:83]: Is this correct to check `authApiEndpoint` and then overwrite `apiEndpoint`?

Comment (cemreyuksel) [reply apps/commercetools-without-search/src/config.ts]: No, this is copy paste error. Pushed the right commit now, sorry.

**Review by mrlaessig — APPROVED** [review]

**Review by matthew-gordon — APPROVED** [review]
LGTM, feel free to merge 🚀

**Pull request merged**